helm/caicloud-event-exporter-app/templates/install.yaml

```diff
@@ -3,6 +3,13 @@ kind: ServiceAccount
 metadata:
   labels:
     {{- include "labels.common" . | indent 4 }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ include "name" . }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    giantswarm.io/service-type: managed
+    helm.sh/chart: {{ include "chart" . }}
```

Comment:
Wait, this is added by helm deploy time, right? Same as `instance` and `managed-by`? It is the same for everything below.

Reply:
These are part of the [`hack/labels-transformer`](https://github.com/giantswarm/caicloud-event-exporter-app/blob/main/hack/labels-transformer.yaml) and now you mention it, I'm beginning to wonder if that file has any value.